.travis.yml

```diff
@@ -28,7 +28,9 @@ before_install:
 
 script:
  - ./adop
- - travis_wait 40 ./adop compose init
+ - while sleep 9m; do echo "=====[ $SECONDS seconds still running ]====="; done &
```

Comment:
9 minute wait? Can we do a loop every 10 seconds and exit when the service is ready? A flat 9m could be to short, could be way to long.

Reply:
That is an interesting problem.

 - It looks like TravisCI provisions 2cpu / 7.5 mem for linux build ENVs. (https://docs.travis-ci.com/user/reference/overview/)
 - GitLab looks to req. 1CPU+4Gb as a base level. (https://docs.gitlab.com/ce/install/requirements.html)
 - TravisCI also has a max. execution limit of around 30 minutes as per the build logs (https://travis-ci.org/Accenture/adop-docker-compose/builds/465007061?utm_source=github_status&utm_medium=notification). Ideally build/test steps should be < 10 minutes. I `feel` like a different resolution is needed here other than increasing the build timeout. Maybe use a pre-build TravisCI container image?